plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/pages/JavadocContentNodes.kt

```diff
@@ -12,12 +11,16 @@ enum class JavadocContentKind : Kind {
 abstract class JavadocContentNode(
     dri: Set<DRI>,
     kind: Kind,
-    override val sourceSets: Set<DokkaSourceSet>
+    override val sourceSets: Set<ContentSourceSet>
 ) : ContentNode {
     override val dci: DCI = DCI(dri, kind)
     override val style: Set<Style> = emptySet()
     override val extra: PropertyContainer<ContentNode> = PropertyContainer.empty()
     override fun withNewExtras(newExtras: PropertyContainer<ContentNode>): ContentNode = this
+
+    // TODO: Support needed?
```

Comment:
I think we should implement support for this, but data classes would really help here 🙄

Reply:
Done ✅